src/effects/audiounits/AudioUnitEffect.cpp

```diff
@@ -1911,13 +1844,7 @@ bool AudioUnitEffect::SavePreset(const RegistryPath & group) const
 
    // Now retrieve the preset content
    CF_ptr<CFPropertyListRef> content;
-   UInt32 size = sizeof(content);
-   AudioUnitGetProperty(mUnit.get(),
-                        kAudioUnitProperty_ClassInfo,
-                        kAudioUnitScope_Global,
-                        0,
-                        &content,
-                        &size);
+   GetFixedSizeProperty(kAudioUnitProperty_ClassInfo, content);
```

Comment:
maybe we could add a check on the return value?

Reply:
Yes, I will rewrite the commit for getting fixed sized properties to check return statuses always, here and elsewhere